cmd/agent/main.go

```diff
@@ -49,10 +51,12 @@ func main() {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "error setting klog flags: %v", err)
 	}
-	local.VisitAll(func(fl *flag.Flag) {
-		fl.Name = util.Normalize(fl.Name)
-		flags.AddGoFlag(fl)
-	})
+	flags.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
+	flags.AddGoFlagSet(local)
+
+	logs.InitLogs()
```

Comment:
Can you run `./bin/proxy-agent --help` before and after your change, and provide the diff to this PR? We want to makes sure we aren't clobbering flags.